main.rb

```diff
@@ -1,160 +1,52 @@
 require_relative 'app'
+require_relative 'book_operations'
```

Comment:
- [ ] @Movgto and @ZabihullahNooriWardak Great work implementing the changes requested, however when I run the app I am getting the error below, this is because you are importing the `book_operation` file wrongly.
![image](https://github.com/Movgto/oop-school-library/assets/61408860/a57d54f6-a783-4bb1-b614-8374b52e0655)

- [ ] @Team I also encountered a problem when trying to create a person after fixing the above error as shown below:
![image](https://github.com/Movgto/oop-school-library/assets/61408860/3af956b8-415b-44d6-ba29-7d5891d2d99d)
![image](https://github.com/Movgto/oop-school-library/assets/61408860/6fb1b340-9178-47a6-b7ba-303ef671a6e1)